addOns/fuzz/src/main/resources/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -159,7 +159,6 @@ fuzz.fuzzer.dialog.messagelocations.locations.label = Fuzz Locations:
 fuzz.fuzzer.dialog.messagelocations.button.payloads.label = Payloads...
 fuzz.fuzzer.dialog.messagelocations.button.processors.tooltip = Allows to manage the payloads of the selected location.
```

Comment:
`processors` → `payloads` and tooltip of `payloadsButton` in `FuzzMessageLocationsPanel` needs to be updated.